recipes-core/udev/udev-rules-udisks-rpi_1.0.bb

```diff
@@ -0,0 +1,11 @@
+DESCRIPTION = "add udisk/udev rule to hide boot partition from udev"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+SRC_URI = "file://80-udisks-rpi.rules"
+
+do_install () {
+	install -d ${D}${base_libdir}/udev/rules.d
+	install -m 644 ${WORKDIR}/80-udisks-rpi.rules ${D}${base_libdir}/udev/rules.d
```

Comment:
@hhromic I suggested this as well but the problem is that users might want the disk udev rules but not these ones (or the other way around). On a second thought, we could use different packages of the same recipe.

Reply:
Ah I see.
After thinking more about this, I believe the question starts to go down to *what is essential to meta-raspberrypi and what is just convenience?*. There must be a clear separation between that in this layer.
The `meta-raspberrypi` layer should be aimed to be very essential to just make RPI boards build with all the base hardware functionality. This in turn makes the layer easier to maintain.
According to Yocto philosophy, it is supposed users create their own layers on top of other layers to make further customisations and conveniences to their needs.
Is this udev rule essential to make RPI boards boot fully-functional? Hidding the boot partition from desktops is something not only for RPI builds, but any other machine that also has vfat boot partitions (which I know exist). So, it really belongs to meta-raspberrypi?

Reply:
Well, yes and no. It is specific to rpi because we use a specific partition table which is machine dependent. So it's somehow at the boundary in my opinion. And generally, as long as changes are connected to the BSP, I do accept "convenience work" as well.

Reply:
I agree this is in the "gray area". So I guess is fine :)